faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -93,10 +97,9 @@ public Builder withFaunaRoot(URL root) {
     }
 
     /**
-     * Sets the Auth Token that the built {@link Connection} will provide to FaunaDB. This must be provided in order
-     * for client to authenticate with FaunaDB.
+     * Sets the authentication token or key for the {@link Connection} instance.
      *
-     * @param token the auth token.
+     * @param token the auth token or key
      * @return this {@link Builder} object
      */
```

Comment:
Should we add an @see to https://fauna.com/documentation/howto/security

Reply:
That page in particular has very little information about security. There is a ticket open fix it.

I'm a little resistant to add pages rather than the API reference because they could be renamed and we're left with broken links scattered throughout the drivers' code.